.env

```diff
@@ -0,0 +1,4 @@
+MQTT_HOST=localhost
+MQTT_PORT=1883
+MQTT_BASE_TOPIC=oss
+ENVIRONMENT=staging
```

Comment:
Is this meant to be committed? Same for .DS_Store file?

Reply:
Resolved